frontend/src/pages/ArtMap/ArtMap.jsx

```diff
@@ -116,8 +116,10 @@ export default function ArtMap() {
    */
   function onMapLoad(map) {
     const geoControl = new GeolocateControl({ trackUserLocation: true });
+    if (lat && long) {
+      setMapLocation([lat, long])
+    }
```

Comment:
I like the idea of viewing the artwork from the map page! If you want the zoom to do the right thing and avoid adding a new route, you could just pass the artwork id. For example `/map/QXJ0d29ya1R5cGU6RmlyZSBXYWxs` will take you to the artwork on the map with that id.

Reply:
I had initially tried this, but I think it currently just opens up the sidebar for the artwork without changing where the map is looking.